tests/test_system/test_reload.py

```diff
@@ -59,7 +59,7 @@ def send_cmd(s, req, resp):
 
 @with_setup(_setup, _teardown)
 def test_reload_with_old_conf():
-    if nc.version() < '0.4.5':
+    if nc.version() < '0.4.2':
```

Comment:
I think these should be 0.4.6, or anything higher than the current version. Not sure this matters for us since we don’t run these tests.

Reply:
Reading the actual purpose of these, it feels like bumping them endlessly was a mistake, so I reset them back down.

Reply:
👍